jib-plugins-common/src/main/java/com/google/cloud/tools/jib/plugins/common/BuildStepsRunner.java

```diff
@@ -38,12 +39,45 @@
 import java.io.IOException;
 import java.net.UnknownHostException;
 import java.nio.file.Path;
+import java.util.StringJoiner;
 import java.util.concurrent.ExecutionException;
 import org.apache.http.conn.HttpHostConnectException;
 
 /** Runs a {@link BuildSteps} and builds helpful error messages. */
 public class BuildStepsRunner {
 
+  private static final String STARTUP_MESSAGE_PREFIX_FOR_DOCKER_REGISTRY =
+      "Containerizing application to ";
+  private static final String SUCCESS_MESSAGE_PREFIX_FOR_DOCKER_REGISTRY =
+      "Built and pushed image as ";
+
+  private static final String STARTUP_MESSAGE_PREFIX_FOR_DOCKER_DAEMON =
+      "Containerizing application to Docker daemon as ";
+  private static final String SUCCESS_MESSAGE_PREFIX_FOR_DOCKER_DAEMON =
+      "Built image to Docker daemon as ";
+
+  private static final String STARTUP_MESSAGE_FORMAT_FOR_TARBALL =
+      "Containerizing application to file at '%s'...";
+  private static final String SUCCESS_MESSAGE_FORMAT_FOR_TARBALL =
+      "Built image tarball at \u001B[36m%s\u001B[0m";
+
+  private static CharSequence colorCyan(CharSequence innerText) {
+    return new StringBuilder().append("\u001B[36m").append(innerText).append("\u001B[0m");
+  }
+
+  private static String buildMessageWithTargetImageReferences(
+      BuildConfiguration buildConfiguration, String prefix, String suffix) {
+    String targetRegistry = buildConfiguration.getTargetImageConfiguration().getImageRegistry();
+    String targetRepository = buildConfiguration.getTargetImageConfiguration().getImageRepository();
+
+    StringJoiner successMessageBuilder = new StringJoiner(", ", prefix, suffix);
+    for (String tag : buildConfiguration.getAllTargetImageTags()) {
+      successMessageBuilder.add(
+          colorCyan(ImageReference.of(targetRegistry, targetRepository, tag).toString()));
```

Comment:
These ANSI sequences don't work for IDEs.  But they can be moved out to the logger once the emitting rolls out.

Reply:
This is actually part of the Maven/Gradle plugins, so this will actually be code outside of Jib Core. Will definitely add startup and success events in Jib Core though.

Reply:
Missed that this was in -plugins-common. 👍